--- config/environment.rb.orig  2012-04-14 17:21:06.000000000 +0900
+++ config/environment.rb       2012-04-18 13:32:43.827484258 +0900
@@ -57,6 +57,8 @@
   config.gem 'rubytree', :lib => 'tree'
   config.gem 'coderay', :version => '~>1.0.0'

+  config.action_controller.session = { :key => "_redmine_session", :secret => "2311f9eceb03a10d3c93ff10dd04e38e" } 
+
   # Load any local configuration that is kept out of source control
   # (e.g. gems, patches).
   if File.exists?(File.join(File.dirname(__FILE__), 'additional_environment.rb'))
